lib/pacemaker/pcmk_sched_allocate.c

```diff
@@ -1392,7 +1392,9 @@ stage5(pe_working_set_t * data_set)
     for (; gIter != NULL; gIter = gIter->next) {
         pe_node_t *node = (pe_node_t *) gIter->data;
 
-        dump_node_capacity(log_prio, "Original", node);
+        if (pcmk_is_set(data_set->flags, pe_flag_show_utilization)) {
+            out->message(out, "node-capacity", node, "Original");
+        }
```

Comment:
The lost piece is being able to trace-log the message via the daemons

Reply:
I had hoped that using a log formatter here would allow the messages to be logged, though I see it would not be at the appropriate level.  I could potentially introduce a new message for this kind of thing, or even a whole new daemon-specific logger.  Trying to sort out all the different printing/logging needs of the daemons and command line tools is difficult.

I've also not tested the before and after to see what is actually happening.  I guess that's the next thing to do.

Reply:
The daemons don't set pe_flag_show_utilization, though they could if this custom message knew to use trace level when using the logger formatter.

One possibility would be to keep the log level in the log formatter's private data, and have a formatter-specific function to set it.

Reply:
Done - I've added a patch to allow setting log level and set the flags in the daemons so these output messages will be called.